src/integrationtest/docker/tomcat/conf.yml

```diff
@@ -0,0 +1,20 @@
+packaging:
+  add:
+    - target/addon-self-administration-${project.version}
```

Comment:
This is the copy task for the assets: https://github.com/osiam/addon-self-administration/pull/170/files#diff-600376dffeb79835ede4a0b285078036R521